src/frontend/src/components/common/Footer.tsx

```diff
@@ -0,0 +1,18 @@
+import React from 'react'
+
+function Footer() {
+  return (
+    <footer>
+        <ul>
+            <li><a>Home</a></li>
+            <li><a>About gov.bc.ca</a></li>
+            <li><a>Disclaimer</a></li>
+            <li><a>Accessiblility</a></li>
+            <li><a>Copyright</a></li>
+            <li><a>Contact us</a></li>
+        </ul>
+    </footer>
+  )
+}
+
+export default Footer;
```

Comment:
Missing empty line at the end.

Reply:
Thanks, I also caught one in the Header component.